Configuration/Settings.yaml

```diff
@@ -34,6 +34,15 @@ Neos:
           'Neos.Neos.UI:HostOnlyStyles':
             resource: '${"resource://" + Neos.Ui.StaticResources.compiledResourcePackage() + "/Public/Styles/HostOnlyStyles.css"}'
             position: 'start 900'
+          'WorkSans:200':
```

Comment:
@dimaip We switched because of this, I think: https://github.com/neos/brand/blob/2f2fe306f0ce1c5ee2ca7345c51464134cfad247/index.js#L23

Is it possible, that the brand assets are used for both, the website and the UI?